prereise/gather/flexibilitydata/geo/batch_process.py

```diff
@@ -0,0 +1,68 @@
+import os
+import urllib.request
+
+from doe import doe_data
+from geo import geo_data
+
+
+def get_cache_from_blob(rel_cache_path="..\\cache"):
+    """download previously uploaded cached files from BLOB storage
```

Comment:
download --> Download. Also should it be: `Download files from BLOB storage` since it does not matter to the user that the files have been uploaded at some point.